safe/reportv4/impact_report.py

```diff
@@ -281,6 +281,8 @@ def __init__(
         self._aggregation_impacted = (
             aggregation_impacted or
             self._impact_function.aggregation_impacted)
+        if extra_layers is None:
```

Comment:
After some thought again, I think it's fine to do like this. The method expect a list if it gets an argument. If it doesn't get the argument `extra_layers`, it will use `None` (or any mutable value) and replace it with empty list in the code. I don't think we should handle the case where the argument is not a list, just like we did with the other arguments.